README.md

```diff
@@ -34,7 +34,7 @@ This Berbix Go library provides simple interfaces to interact with the Berbix AP
 
 ### `Client`
 
-##### NewClient(secret string, options *ClientOptions) Client
+##### NewClient(secret string, options \*ClientOptions) Client
```

Comment:
why the `\`?

Reply:
My guess is that Yifeng has a markdown auto-formatter that automatically escapes unclosed formatting tags. The `\` shouldn't get rendered by standard markdown renderers.